driver/main_args.ml

```diff
@@ -167,9 +167,16 @@ let mk_inline_toplevel f =
 
 let mk_inlining_report f =
   "-inlining-report", Arg.Unit f, " Emit `.<round>.inlining' file(s) (one per \
-      round) showing the inliner's decisions"
+      round) showing the inliner's decisions, in a human readable format"
 ;;
 
+let mk_inlining_report_serialized f =
```

Comment:
Let's not use "serialized" as the spelling differs between UK/US English.  What about "`-inlining-report-bin`"?

Reply:
changed to `-inlining-report-bin`